programs/stake/src/points.rs

```diff
@@ -21,13 +13,6 @@ pub struct PointValue {
     pub points: u128, // over these points
 }
 
-#[derive(Debug, PartialEq, Eq)]
-pub(crate) struct CalculatedStakePoints {
-    pub(crate) points: u128,
-    pub(crate) new_credits_observed: u64,
-    pub(crate) force_credits_update_with_skipped_reward: bool,
-}
-
 #[derive(Debug)]
 pub enum InflationPointCalculationEvent {
```

Comment:
Same question as above for "InflationPointCalculationEvent" and "SkippedReason"?

Reply:
Same answer as above applies here